### PR TITLE
[metal3] Link disk & nic inventory items to tabs

### PR DESCRIPTION
**Pull request by honza**

### CODE REVIEW

File: frontend/public/components/dashboard/inventory-card/inventory-item.tsx [hunk 1]
```diff
@@ -53,9 +53,14 @@ export const InventoryItem: React.FC<InventoryItemProps> = React.memo(
     } else {
       status = children;
     }
+    let label = <>{isLoading || error ? title : `${count} ${title}`}</>;
+
+    if (link) {
+      label = <Link to={link}>{label}</Link>;
```
Comment (jtomasek) [inline frontend/public/components/dashboard/inventory-card/inventory-item.tsx]: The link capability is fairly limiting here, although it probably covers current requirements. `<Link>`'s `to` prop is not limited to just string.

Comment (knowncitizen) [reply frontend/public/components/dashboard/inventory-card/inventory-item.tsx]: Yeah for this i think it's fine

**Comment by rawagner** [comment]
implemented as a part of https://github.com/openshift/console/pull/3149 . I think you can close this one

**Pull request closed without merging**